Automattic/wp-calypso
# Fetch the language revision file on bootstrapping

## Summary
This PR is based on https://github.com/Automattic/wp-calypso/pull/26973. It introduces a new server-side module, `i18n-bootstrap`, which contains the function `getLangRevisions()` for fetching the language revision map file along side `user-bootstrap`.

## Test Plan
~1. Refer to code-D17741 for instructions of the backend preparation.~ This is no longer needed since it has been deployed.

1. `npm run build-docker`
1. `npm run docker`
1. Access your dockerized calypso according to your host setup. In my case, it is http://wpcalypso.wordpress.com.
1. Open the HTML inspector. The language file should have the revision number appended as the cache buster. e.g. `<script src="//widgets.wp.com/languages/calypso/zh-tw.js?v=35623"></script>`
1. In your browser console, execute `window.langRevisions` should print the revision map, indicating it has been inlined properly.
1. Switch your interface language and make sure the above stay true.
1. Undo the backend preparation from step 1, and see that the language file is loaded without the cache buster.
